cli/.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgres://localhost:5432/knowledge
```

Comment:
I have my `.envrc` file in repo root, and I have that added to the `.gitignore` too!

That way `git` won't bother you about changes to that file or anything either!